lib/v2/ieee/latestdate.js

```diff
@@ -0,0 +1,102 @@
+const got = require('@/utils/got');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const punumber = ctx.params.journal;
+    const sortType = ctx.params.sortType || 'vol-only-seq';
+    const hostUrl = 'https://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=' + punumber;
+    const headers = {
+        Host: 'ieeexplore.ieee.org',
```

Comment:
Why specified `ieeexplore.ieee.org` again in `Host` headers when `ieeexplore.ieee.org` is already specified in `hostUrl`?

Reply:
Thanks your attention, I have modified it.
```javascript
const config = require('@/config').value;
const headers = {
    Accept: 'application/json, text/plain, */*',
    'Accept-Encoding': 'gzip, deflate, br',
    'Accept-Language': 'zh-CN,zh;q=0.9',
    Host: host.replace('https://', ''),
    'User-Agent': config.ua,
    Referer: hostUrl,
};
```